src/databricks/labs/ucx/runtime.py

```diff
@@ -130,7 +130,7 @@ def assess_pipelines(cfg: WorkspaceConfig):
     crawler.snapshot()
 
 
-@task("assessment")
+@task("assessment", is_azure=True, is_aws=False, is_gcp=False)
```

Comment:
Can we have is_azure=True only ? It allows for less boilerplate and easier readability

Reply:
@william-conti That was considered in the first implementation.. that led to more complex logic evaluation, a bug and made it difficult to evaluate correctness. This is only needed on cloud specific tasks (1 current, 1 planned).

Reply:
Why ? IMO you need to modify cloud_compatible method to compare the current task key and compare it with the conf and done

Reply:
fully agree with @william-conti . i'd suggest a string instead of boolean flags - that how we do it in go/openapi/spec, at least ;)

Reply:
@nfx I can't find the reference on the page.